src/main/java/org/folio/sidecar/integration/users/UserService.java

```diff
@@ -12,22 +12,29 @@
 import io.vertx.ext.web.client.HttpResponse;
 import io.vertx.ext.web.client.WebClient;
 import jakarta.enterprise.context.ApplicationScoped;
-import lombok.RequiredArgsConstructor;
+import jakarta.inject.Named;
 import lombok.extern.log4j.Log4j2;
 import org.folio.sidecar.integration.users.configuration.property.ModUsersProperties;
 import org.folio.sidecar.integration.users.model.User;
 import org.folio.sidecar.service.ServiceTokenProvider;
 
 @Log4j2
 @ApplicationScoped
-@RequiredArgsConstructor
 public class UserService {
 
-  private final WebClient webClient;
+  private final WebClient webClientTls;
   private final ModUsersProperties modUsersProperties;
   private final Cache<String, User> userCache;
   private final ServiceTokenProvider serviceTokenProvider;
 
+  public UserService(@Named("webClientTls") WebClient webClientTls, ModUsersProperties modUsersProperties,
```

Comment:
you can add `@Named` annotation to the `lombok.config` and continue using `@RequiredArgsConstructor`

Reply:
> you can add `@Named` annotation to the `lombok.config` and continue using `@RequiredArgsConstructor`

No need to do that, Intellij idea provides a warning for this case:
`Private access modifier usage is not recommended for Quarkus CDI members`